_posts/2017-01-24-early-adoption-of-marketplace-pays-off.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Early adoption of Marketplace pays off"
+category: [news]
+tag: 'digital marketplace'
+thumbnail: /images/blog-thumbnails/modernising-mygov-thumb.png
+hero-image: /images/blog-content/modernising-mygov-hero.png
+feed: true
+---
+
+# Early adoption of Marketplace pays off
+
+Local councils are starting to take advantage of the speed and ease of procurement that the Digital Marketplace brings.
+ 
+The City of Casey in Victoria published its first brief just eight days after our Beta release - an individual specialist opportunity for a developer experienced in Gravity forms. 
+ 
+In this interview, the Manager of Digital for City of Casey, Chris O'Connor, explains how happy he was to find the experience simple, clear and fast - even it its earliest Beta stage.
+ 
+The Digital Marketplace has been created so buyers from all levels of government can access digital expertise quickly and easily. 
+ 
+Currently there are 252 registered government buyers from 78 different agencies, including the Australian Tax Office, the ABC, the Department of Health and Department of Finance, Services and Innovation to name a few.
+
+## Watch the video
+
+<div class="youtube-embed">
+  <iframe width="100%" height="400" src="https://youtu.be/c-KQqtmgm-I" frameborder="0" allowfullscreen></iframe>
+  
+  <details open data-label="content-accordion-1-example" aria-expanded="false">
+    <summary><h3>Read transcript - 2:21 min</h3></summary>
+    <div class="accordion-panel" markdown="1">
+    
+*Caption: Interview with Chris O'Connor the Manager of Digital, City of Casey - one of the first buyers in the Digital Marketplace.*
```

Comment:
Move the caption after the last `</div>` so that it appears after teh "Read Transcript accordion"